docs/reference/client-config.md

```diff
@@ -32,10 +32,10 @@ JSON keys to learn more about the configuration section corresponding to that ke
   <a href="#trust_dir-section-optional">"trust_dir"</a> : "~/.docker/trust",
   <a href="#remote_server-section-optional">"remote_server"</a>: {
     "url": "https://my-notary-server.my-private-registry.com",
-    "root-ca": "./fixtures/root-ca.crt",
+    "root_ca": "./fixtures/root-ca.crt",
```

Comment:
if you don't mind including in this PR: could we also update the `root-ca` entries in the `remote_server section` and table below?